src/main/webapp/resources/index.html

```diff
@@ -1,56 +1,76 @@
 <!doctype html>
 <html ng-app="calendar">
 <head>
-    <title>calendarPLUS</title>
     <link rel="stylesheet" type="text/css" href="./bower_components/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="./bower_components/fullcalendar/fullcalendar.css"/>
     <link rel="stylesheet" type="text/css" href="./resources/styles/style.css">
     <link rel="shortcut icon" type="image/x-icon" href="./resources/images/favicon.ico"/>
-    <title>{{title? title: "PLUS"}</title>
-    <link rel="stylesheet" type="text/css" href="/bower_components/bootstrap/dist/css/bootstrap.min.css">
-    <!--UI CALLENDAR-->
-    <link rel="stylesheet" href="/bower_components/fullcalendar/fullcalendar.css"/>
-    <script type="text/javascript" src="/bower_components/jquery/dist/jquery.js"></script>
-    <script type="text/javascript" src="/bower_components/jquery-ui/ui/jquery-ui.js"></script>
-    <script type="text/javascript" src="/bower_components/angular/angular.js"></script>
-    <script type="text/javascript" src="/bower_components/angular-ui-calendar/src/calendar.js"></script>
-    <script type="text/javascript" src="/bower_components/fullcalendar/fullcalendar.js"></script>
-    <script type="text/javascript" src="/bower_components/fullcalendar/gcal.js"></script>
-    <script src="/bower_components/angular-bootstrap/ui-bootstrap-tpls.min.js"></script>
+    <title>{{title? title: "PLUS"}}</title>
 </head>
 
 <body>
 <div>
-    <nav class="navbar navbar-inverse" role="navigation" style="margin-bottom: 0">
-        <div class="navbar-header" ng-click="title = ''">
-            <a class="navbar-brand" href="#"><span class="glyphicon glyphicon glyphicon-calendar"></span></a>
+    <nav id="navigation" class="navbar navbar-inverse" role="navigation">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="#/"><span class="glyphicon glyphicon glyphicon-calendar"></span></a>
         </div>
         <ul class="nav navbar-nav">
             <li><a href="#calendar">Calendar</a></li>
             <li><a href="#map">Map</a></li>
         </ul>
     </nav>
-    <ol ng-if="title" class="breadcrumb">
-        <li><label>{{title}}</label></li>
-    </ol>
 </div>
 
-<div class="container">
-    <div ng-view></div>
+<div ng-controller="RootController">
+    <div id="breadcrumb" ng-if="title" class="breadcrumb" style="margin-bottom: 0">
+        <label>{{title}}</label>
+        <span class="pull-right"><a href="" ng-click="panel.show = !panel.show">{{panel.show? "Hide events panel":
+            "Show events panel"}}</a></span>
+    </div>
+    <div class="no-padding" ng-class="panel.show? 'col-md-8': 'col-md-12'" ng-view></div>
+    <div class="col-md-4 no-padding page-slide" ng-show="panel.show" full-height
```

Comment:
I would rather change width of panel to col-md-3. On the smaller screen it still looks nice